about.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container-md d-flex flex-column align-items-center my-5">
+      <h1
+        class="about-header display-2 fw-bolder text-uppercase text-center my-2"
+      >
+        "Jah Malet Fetari" International Chess Tournament 2023
+      </h1>
+      <div class="sub-heading text-center my-2">
+        <p class="fs-5">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat
+          dignissimos dolore recusandae consectetur iste, sapiente eaque harum
+          impedit dolores corrupti ab minus quis, molestiae excepturi sit cumque
+          tempora iure. Explicabo impedit dolores magnam dicta unde ea eius
+          error! Dolorem similique fuga molestiae eveniet atque praesentium
+          doloribus libero nobis totam tempore.
+        </p>
+      </div>
+      <div class="w-50 my-4">
+        <p class="fs-4 text-center">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Explicabo,
+          doloribus!
+        </p>
+      </div>
+      <div class="about-section-2 d-flex flex-column align-items-center">
+        <h3 class="text-center">
```

Comment:
- [ ] Kindly  😃 try to make the content of tags to be like the Behance as you see here below the text there is a link,so please try to make that like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225008723-3485a1ec-86df-4b00-b796-c80a97aebe42.png)  |    ![image](https://user-images.githubusercontent.com/83360636/225008818-14222cf1-fa69-440d-bee9-a29e2ff8e555.png)   |